### PR TITLE
- add max file size

### PR DESCRIPTION
**Pull request by EwelinaSkrzypacz**
In this pr I added to `php.ini` and `nginx.conf` max size of files

### CODE REVIEW

File: environment/dev/app/php.ini [hunk 1]
```diff
@@ -1,5 +1,7 @@
 [PHP]
 memory_limit = 256M
+upload_max_filesize = 3M
+post_max_size = 3M
```
Comment (mtracz) [inline environment/dev/app/php.ini:4]: Why exactly 3?
Is it related to uploaded files?
Maybe we should increase this a little bit more?

**Comment by krzysztofrewak** [comment]
What happened? As far as I don't have no problem with changing things like this, I'm little worried about 3MB photos on website.

**Comment by EwelinaSkrzypacz** [comment]
> What happened? As far as I don't have no problem with changing things like this, I'm little worried about 3MB photos on website.

Kasia had a problem with uploading photo ~1.5 MB

**Comment by EwelinaSkrzypacz** [comment]
> Defaults: PHP:
> 
> * post_max_size = 8M
> * upload_max_filesize = 2M
> 
> Nginx:
> 
> * client_max_body_size = 1m;
> 
> So it was blocked by Nginx. I think we can bump `client_max_body_size` to `2m`.

Done. :)

**Review by mtracz — APPROVED** [review]

**Review by krzysztofrewak — APPROVED** [review]

**Comment by EwelinaSkrzypacz** [comment]
> We can consider to upload images already converted to `.webp` format if we want to reduce size. But this will require additional work for moderators of website. There are online converters so I think it is not big deal.

I think about converting images to .webp automatically during uploading image, but I don't know how time-consuming to make this will be

**Pull request merged**